bedrock/firefox/templates/firefox/tiles.html

```diff
@@ -14,8 +14,15 @@
    3. Breadcrumbs ("Home > Title")
    3. Page title (large headline)
 #}
-{% block page_title %}{{ _('A Brand New Tiles Experience') }}{% endblock %}
 
+{% block page_title -%}
+{%- if l10n_has_tag('bug_1124724_copy_update') -%}
```

Comment:
No need to have a tag this detailed, `tiles_update` is enough. Also, the comment is not necessary (`L10n: This sentence is obsolete. Do not translate.`).